### PR TITLE
:memo: Add section on paused environments

### PR DESCRIPTION
**Pull request by AnouckColson**
## Why

Included in the Git 20 release: Development environments that haven't been used for 15 days will now be paused automatically. They can be resume by users themselves (through a redeployment of the environment).

## What's changed

I added a section to the [Manage environments](https://docs.platform.sh/environments.html) page about this feature.

### CODE REVIEW

**Review by chadwcarlson — DISMISSED** [review]

File: docs/src/environments/_index.md [hunk 1]
```diff
@@ -186,3 +186,35 @@ Staging
...
     Ticket-395
 ```
+
+## Paused environments
+
+Development environments are often used for a limited time and then abandoned.
+To prevent unnecessary consumption of resources,
+Platform.sh automatically pauses development environments where code hasn't been pushed in 15 days.
+
+You can resume a paused environment at any time.
```
Comment (remilejeune2) [inline docs/src/environments/_index.md:196]: We could explain that "Pushing new code automatically resume the environment, which can also be resume at any time"

When you are push

File: docs/src/environments/_index.md [hunk 2]
```diff
@@ -186,3 +186,72 @@ Staging
...
+## Paused environments
+
+Development environments are often used for a limited time and then abandoned.
+To prevent unnecessary consumption of resources,
+Platform.sh automatically pauses development environments where code hasn't been pushed in 14 days.
+
+You can also pause an environment manually at any time.
+
+Currently, you can only pause and resume environments through the [Console](../administration/web/_index.md)
+and [Platform.sh API](https://api.platform.sh/).
```
Comment (chadwcarlson) [inline docs/src/environments/_index.md:199]:
```suggestion
and the [Platform.sh API](https://api.platform.sh/).
```

File: docs/src/environments/_index.md [hunk 3]
```diff
@@ -186,3 +186,72 @@ Staging
...
+
+<--->
++++
+title=Using the API
++++
+
+Run the following command:
+
+```bash
+platform p:curl /environments/{{< variable "ENVIRONMENT_ID" >}}/pause -X POST -p {{< variable "PROJECT_ID" >}}
```
Comment (chadwcarlson) [inline docs/src/environments/_index.md:224]: @AnouckColson wasn't there a PR recently that addressed that we want to use expanded commands (`project:curl` vs `p:curl`) preferably? What's our standard?

File: docs/src/environments/_index.md [hunk 4]
```diff
@@ -186,3 +186,72 @@ Staging
...
+
+<--->
++++
+title=Using the API
++++
+
+Run the following command:
+
+```bash
+platform p:curl /environments/{{< variable "ENVIRONMENT_ID" >}}/resume -X POST -p {{< variable "PROJECT_ID" >}}
```
Comment (chadwcarlson) [inline docs/src/environments/_index.md:252]: Same here

**Review by chadwcarlson — DISMISSED** [review]

File: docs/src/environments/_index.md [hunk 5]
```diff
@@ -186,3 +186,81 @@ Staging
...
+This includes your future production environment.<BR>
+To prevent your production environment from being paused automatically,
+[upgrade to a non-development plan](https://platform.sh/pricing/).
+
+{{< /note >}}
+
+You can also pause an environment manually at any time.
+Currently, you can only pause and resume environments through the [Console](../administration/web/_index.md)
+and the [Platform.sh API](https://api.platform.sh/).
+In the future, you'll be able to do so in the [CLI](../administration/cli/_index.md) too.
```
Comment (remilejeune2) [inline docs/src/environments/_index.md:209]: We are in the future. `pause` and `resume` are available in the CLI:
```
  environment:pause                                              Pause an environment
  environment:resume                                             Resume a paused environment
```

File: docs/src/environments/_index.md [hunk 6]
```diff
@@ -186,3 +186,81 @@ Staging
...
+
+<--->
++++
+title=Using the API
++++
+
+Run the following command:
+
+```bash
+platform project:curl /environments/{{< variable "ENVIRONMENT_ID" >}}/pause -X POST -p {{< variable "PROJECT_ID" >}}
```
Comment (remilejeune2) [inline docs/src/environments/_index.md:233]: platform environment:pause -p {{< variable "PROJECT_ID" >}} -e {{< variable "ENVIRONMENT_ID" >}}

File: docs/src/environments/_index.md [hunk 7]
```diff
@@ -186,3 +186,81 @@ Staging
...
+
+<--->
++++
+title=Using the API
++++
+
+Run the following command:
+
+```bash
+platform project:curl /environments/{{< variable "ENVIRONMENT_ID" >}}/resume -X POST -p {{< variable "PROJECT_ID" >}}
```
Comment (remilejeune2) [inline docs/src/environments/_index.md:261]: platform environment:resume -p {{< variable "PROJECT_ID" >}} -e {{< variable "ENVIRONMENT_ID" >}}

File: docs/src/environments/_index.md [hunk 8]
```diff
@@ -186,3 +186,81 @@ Staging
...
+
+## Paused environments
+
+Development environments are often used for a limited time and then abandoned.
+To prevent unnecessary consumption of resources,
+Platform.sh automatically pauses development environments where code hasn't been pushed in 14 days.
+
+{{< note >}}
+
+If you're on a Professional (development) plan,
```
Comment (remilejeune2) [inline docs/src/environments/_index.md:198]: Is a "development" Plan tied to the Professional tier?

**Review by remilejeune2 — CHANGES_REQUESTED** [review]

**Review by remilejeune2 — APPROVED** [review]

**Pull request merged**